README.md

```diff
@@ -7,6 +7,49 @@ AwesomeBits
 Configuration
 -------------
 
+### Docker
+
+To start developing this project quickly, feel free to use the [docker-compose](https://docs.docker.com/compose/) workflow. This will install a local Postgres database in a Docker container, and the Rails app in another container.   
+
+#### Installing the Docker runtime
+
+##### Linux: Install the Native Docker Engine
+
+If you're developing on a Linux system, follow [these instructions](https://docs.docker.com/installation/) to install Docker natively on your system. You will also need to install [Docker Compose](https://docs.docker.com/compose/install/).
+
+##### Mac or Windows: Install Docker Machine
+
+If you're developing from a Mac or Windows machine, you will need to use [Docker Machine](https://docs.docker.com/machine/) to run the Docker engine in a Linux VM and proxy commands to it. You can install Docker Machine via the [Docker Toolbox](https://www.docker.com/toolbox). The Docker toolbox will also install [Docker Compose](https://docs.docker.com/compose/).
+
+Once you have Docker Machine installed, you'll need to create a new machine for this project. Assuming you have [Virtualbox](https://www.virtualbox.org/wiki/Downloads) installed, you'd run the following:
+
+```bash
+docker-machine create -d virtualbox awesomebits
+eval $(docker-machine env awesomebits)
+```
+
+#### Starting the app with Docker Compose
+
+Once you have the Docker engine and [Docker Compose](https://docs.docker.com/compose/install/) installed, you should be able to launch the app with the following command:
+
+```bash
+docker-compose up
+```
+
+If successful, your terminal should look something like this:
+
+![Docker Compose Up Success](doc/awesomebits-docker-compose-up.gif)
+
+#### Executing arbitary commands with Docker Compose
+
+You can also run arbitrary commands within the Docker container. For example, to run `rails console`, do the following:
+
+```bash
+docker-compose run web rails console
+```
+
+![Running Rails Console with Docker Compose](doc/awesomebits-docker-compose-rails-console.gif)
+
```

Comment:
I plan on recording a screencast with more detail on the development cycle over the weekend. The workflow might be better shown than described in text form.